libcontainer/cgroups/systemd/v1.go

```diff
@@ -72,11 +72,13 @@ func genV1ResourcesProperties(c *configs.Cgroup, conn *systemdDbus.Conn) ([]syst
 	var properties []systemdDbus.Property
 	r := c.Resources
 
-	deviceProperties, err := generateDeviceProperties(r.Devices)
-	if err != nil {
-		return nil, err
+	if !c.SkipDevices {
+		deviceProperties, err := generateDeviceProperties(r.Devices)
+		if err != nil {
+			return nil, err
+		}
+		properties = append(properties, deviceProperties...)
 	}
```

Comment:
While I do understand why you changed this, please be aware that disabling this doesn't make systemd not do anything with the devices cgroup, it will actually place it into allow-everything mode.

Reply:
Thank you, fixed.

Reply:
So, if the default is deny-all and a parent cgroup's ebpf is consulted first before the child cgroup, this apparently (
In view of https://github.com/kubernetes/kubernetes/pull/92862#pullrequestreview-444255169) results in no devices accessible.

I guess I'll have to revert to the original hunk here, leaving systemd unconfigured wrt devices, which should result in it using `DevicePolicy=auto`.